src/client/Client.cc

```diff
@@ -15896,17 +15902,23 @@ int Client::ll_commit_blocks(Inode *in,
 
 int Client::ll_write(Fh *fh, loff_t off, loff_t len, const char *data)
 {
+  RWRef_t mref_reader(mount_state, CLIENT_MOUNTING);
+  if (!mref_reader.is_state_satisfied()) {
```

Comment:
mref_reader in the update path?

Reply:
no, no changes to that, there are the changes made here that looks mref_reader in update path:

```
  ldout(cct, 3) << "ll_write " << fh << " " << fh->inode->ino << " " << off <<
    "~" << len << dendl;
  tout(cct) << "ll_write" << std::endl;
  tout(cct) << (uintptr_t)fh << std::endl;
  tout(cct) << off << std::endl;
  tout(cct) << len << std::endl;
```
existed before the checks so moved it after the two checks (making it same as `ll_read`)

AND

added braces to guard `if (!mref_reader.is_state_satisfied())`

Reply:
ACK.